### PR TITLE
Add extra scoped blocks and template hooks

### PR DESCRIPTION
**Pull request by SegiNyn**

### CODE REVIEW

File: indico/modules/events/timetable/templates/balloons/_balloon_utils.html [hunk 1]
```diff
@@ -24,15 +24,17 @@
...
-        {% if obj.venue_name %}
-            <dt><i class='icon-location'></i></dt>
-            <dd>
-                <div>{{ obj.room_name }} ({{ obj.venue_name }})</div>
-                {% if obj.address %}
-                    <div class="address">{{ obj.address }}</div>
-                {% endif %}
-            </dd>
-        {% endif %}
+        {% block venue_info_baloon scoped %}
```
Comment (ThiefMaster) [inline indico/modules/events/timetable/templates/balloons/_balloon_utils.html:27]:
```suggestion
        {% block venue_info_balloon scoped %}
```

File: indico/modules/events/templates/management/_create_event_button.html [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
-    </a>
-    <ul class="i-dropdown">
-        <li>{{ create_event_link(category, 'lecture', _('Lecture')) }}</li>
-        <li>{{ create_event_link(category, 'meeting', _('Meeting')) }}</li>
-        <li>{{ create_event_link(category, 'conference', _('Conference')) }}</li>
-    </ul>
+    {% block create_event_button scoped %}
+        <a href="" class="{{ classes }} i-button arrow js-dropdown"
+        {% if with_tooltip %}title="{% trans %}Create event{% endtrans %}"{% endif %}
+        data-toggle="dropdown">
```
Comment (ThiefMaster) [inline indico/modules/events/templates/management/_create_event_button.html:5]: the indentation is rather weird

Comment (SegiNyn) [reply indico/modules/events/templates/management/_create_event_button.html]: I've added back the indentation

File: indico/modules/events/management/templates/_settings.html [hunk 3]
```diff
@@ -60,6 +60,7 @@
                     {% endif %}
                 {% endcall %}
             </dd>
+            {{ template_hook('extra-event-section-data', event=event) }}
```
Comment (ThiefMaster) [inline indico/modules/events/management/templates/_settings.html:63]:
```suggestion
            {{ template_hook('extra-event-settings-section-data', event=event) }}
```

Comment (ThiefMaster) [reply indico/modules/events/management/templates/_settings.html:63]: An even nicer alternative would be this (UNTESTED):

```patch
--- a/indico/modules/events/management/templates/_settings.html
+++ b/indico/modules/events/management/templates/_settings.html
@@ -21,6 +21,7 @@
         <div class="icon {{ icon }}"></div>
         <div class="text">
             {{ caller() }}
+            {{ template_hook('event-settings-section-extra', event=event, name=name) }}
         </div>
         <div class="toolbar">
             <button class="i-button borderless text-color icon-only icon-edit hide-if-locked"

```

That way you can add new stuff at the end of any section, so if you need to extend another one you don't need to send another PR

Comment (SegiNyn) [reply indico/modules/events/management/templates/_settings.html]: Yes this works, thanks!

File: indico/modules/events/management/templates/_settings.html [hunk 4]
```diff
@@ -21,6 +21,7 @@
         <div class="icon {{ icon }}"></div>
         <div class="text">
             {{ caller() }}
+            {{ template_hook('event-settings-section-extra', event=event, name=name) }}
```
Comment (ThiefMaster) [inline indico/modules/events/management/templates/_settings.html:24]: Looking at this again, maybe calling it `section` is better...

```suggestion
            {{ template_hook('event-settings-section-extra', event=event, section=name) }}
```

**Pull request merged**